MAUI/StepProgressBar/getting-started.md

```diff
@@ -0,0 +1,209 @@
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("Segoe-mdl2.ttf", "SegoeMDL2");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Populating step progressbar items
+
+You can use ItemsSource property of SfStepProgressBar to populate the step progressbar items. Hence, you must create a item collection and bind it to the progressbar control.
+
+* Create a simple Observable Collection of item for the step progressbar,
+
+{% tabs %}
+{% highlight c# tabtitle="ViewModel.cs" %}
+
+public class ViewModel
+{
+    /// <summary>
+    /// The Step progress bar item collection.
+    /// </summary>
+    private ObservableCollection<StepProgressBarItem> stepProgressItem;
+
+    /// <summary>
+    /// The Step progress bar item collection.
+    /// </summary>
+    public ObservableCollection<StepProgressBarItem> StepProgressItem
+    {
+        get
+        {
+            return stepProgressItem;
+        }
+        set
+        {
+            stepProgressItem = value;
+        }
+    }
+
+    public ViewModel()
+    {
+        stepProgressItem = new ObservableCollection<StepProgressBarItem>();
+        stepProgressItem.Add(new StepProgressBarItem() { PrimaryText = "Completed" });
+        stepProgressItem.Add(new StepProgressBarItem() { PrimaryText = "In Progress" });
+        stepProgressItem.Add(new StepProgressBarItem() { PrimaryText = "Not Started" });
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+* Bind the Collection to step progressbar
+
+The Step progressbar control allows you to bind item collection by setting the [ItemsSource]() property from the [SfStepProgressBar](). Bind item collection in both XAML and C#.
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="8" %}
+
+<stepProgressBar:SfStepProgressBar
+                    x:Name="stepProgressBar"
+                    Margin="24,10,0,0"
+                    ActiveStepIndex="1"
+                    ActiveStepProgressValue="60"
+                    LabelSpacing="12"
+                    Orientation="Horizontal"  
+                    ItemsSource="{Binding StepProgressItem}">
+</stepProgressBar:SfStepProgressBar>                                                                                              
+                
+
+<ContentPage.BindingContext>
+    <local:ViewModel />
+</ContentPage.BindingContext>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="9" %}
+
+ViewModel viewModel = new ViewModel();
+SfStepProgressBar stepProgressBar = new SfStepProgressBar()
+{
+    Margin = new Thickness(24, 10, 0, 0),
+    ActiveStepIndex = 1,
+    ActiveStepProgressValue = 60,
+    LabelSpacing = 12,
+    Orientation = StepProgressBarOrientation.Horizontal,
+    ItemsSource = viewModel.StepProgressItem,
+};
+
+this.Content = stepProgressBar;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Active step index value and Active progress value
```

Comment:
Give proper title

Reply:
I have changed the title as **Active step progression**